photon-core/src/main/java/org/photonvision/vision/pipeline/AprilTagPipelineSettings.java

```diff
@@ -32,7 +32,7 @@ public class AprilTagPipelineSettings extends AdvancedPipelineSettings {
     public int numIterations = 40;
     public int hammingDist = 0;
     public int decisionMargin = 35;
-    public boolean doMultiTarget = true;
+    public boolean doMultiTarget = false;
```

Comment:
Why is multitag being disabled?

Reply:
This is just setting the default value, which should match everywhere. This was set to false in the UI by default, which might (?) override this.

Reply:
UI values always get overridden as soon as the backend sends settings. @mcm001 which default value did you want here? You had true in settings and false in UI